src/System.Security.Cryptography.Primitives/tests/CryptoStream.cs

```diff
@@ -162,11 +162,38 @@ public static void NestedCryptoStreams()
         public static void MultipleDispose()
         {
             ICryptoTransform encryptor = new IdentityTransform(1, 1, true);
+
+            using (MemoryStream output = new MemoryStream())
+            {
+                using (CryptoStream encryptStream = new CryptoStream(output, encryptor, CryptoStreamMode.Write))
+                {
+                    encryptStream.Dispose();
+                }
+
+                Assert.Equal(false, output.CanRead);
+            }
+
+#if netcoreapp11
+            using (MemoryStream output = new MemoryStream())
+            {
+                using (CryptoStream encryptStream = new CryptoStream(output, encryptor, CryptoStreamMode.Write, leaveOpen: false))
+                {
+                    encryptStream.Dispose();
+                }
+
+                Assert.Equal(false, output.CanRead);
+            }
+
             using (MemoryStream output = new MemoryStream())
-            using (CryptoStream encryptStream = new CryptoStream(output, encryptor, CryptoStreamMode.Write))
             {
-                encryptStream.Dispose();
+                using (CryptoStream encryptStream = new CryptoStream(output, encryptor, CryptoStreamMode.Write, leaveOpen: true))
+                {
+                    encryptStream.Dispose();
+                }
+
+                Assert.Equal(true, output.CanRead);
             }
+#endif
```

Comment:
Nit: I think elsewhere we've been separating out such tests that depend on new surface area into separate files.